src/main/java/org/kiwiproject/test/dropwizard/app/PostgresAppTestExtension.java

```diff
@@ -0,0 +1,64 @@
+package org.kiwiproject.test.dropwizard.app;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.zonky.test.db.postgres.embedded.LiquibasePreparer;
+import io.zonky.test.db.postgres.junit5.EmbeddedPostgresExtension;
+import io.zonky.test.db.postgres.junit5.PreparedDbExtension;
+import lombok.Getter;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * Multi-purpose extension for {@link Application} testing requiring a PostgreSQL database. The extension will spin up an
+ * embedded PostgreSQL database instance, run the migrations and then configure the {@link DropwizardAppExtension} with the
+ * database information.
+ * @implNote The embedded PostgreSQL extension supports both Flyway and Liquibase, but we are assuming migrations are Liquibase.
```

Comment:
I think `@implNote` is always supposed to be grouped with the other `@xxx` at bottom. Suggest just putting this as a separate paragraph.